libco/mips.c

```diff
@@ -0,0 +1,234 @@
+/*
+  libco.mips (2020-04-20)
+  author: Google
```

Comment:
Google? But the PR title suggests you wrote it?

Reply:
Yes, copyrighted by Google due to my employee status. Clearified copyright/authorship